### PR TITLE
feature: added login page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added Login Page f E-commerce Website

Live Link: https://neog-e-comm-website.netlify.app/

### CODE REVIEW

File: loginstyles.css [hunk 1]
```diff
@@ -1,92 +1,78 @@
...
 body {
     font-family: "Poppins", sans-serif;
 }
-
 .navbar {
-    padding: 0px 150px;
-    height: 80px;
-}
-.navbar input {
-    width: 421px;
```
Comment (irajbhinde) [inline loginstyles.css:11]: Use relative unit - rem instead of px everywhere

File: login.html [hunk 2]
```diff
@@ -19,17 +19,18 @@
...
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
             <input class="search" type="text" placeholder="Search"></input>
 
             <div class="nav-right-container">
-                <button class="btn btn-primary">Login</button>
-                <i class="far fa-heart fa-lg"></i><span class="badge1">4</span>
-                <i class="fas fa-shopping-cart fa-lg"></i><span class="badge2">4</span>
+                <button class="login-btn">Login</button>
```
Comment (HarshnaNHaswani) [inline login.html:31]: for consistent naming, can use btn-login

File: login.html [hunk 3]
```diff
@@ -45,7 +46,7 @@ <h1>Login</h1>
            
             <input class="checkbox" type="checkbox">Remember me</input>
             <a class="forget-password" href="/">Forget Your Password?</a>
-            <button class="btn btn-primary"><a class="login-btn" href="/home/index.html">Log in</a></button>
+            <button class="click-me-link-btn new-ac"><a href="/home/index.html">Log in</a></button>
```
Comment (HarshnaNHaswani) [inline login.html:49]: Unable to understand what's new ac?

File: index.html [hunk 4]
```diff
@@ -19,17 +19,18 @@
...
+            <p class="head">MyWebsite</p>
             <input class="search" type="text" placeholder="Search"></input>
 
             <div class="nav-right-container">
-                <button class="btn btn-primary">Login</button>
-                <i class="far fa-heart fa-lg"></i><span class="badge1">4</span>
-                <i class="fas fa-shopping-cart fa-lg"></i><span class="badge2">4</span>
+                <button class="login-btn">Login</button>
+                <i class="far fa-heart fa-lg"></i>
+                <i class="fas fa-shopping-cart fa-lg"><span class="badge2">4</span></i>
```
Comment (irajbhinde) [inline index.html:33]: Use cursor : pointer for both icons

File: index.html [hunk 5]
```diff
@@ -45,7 +46,7 @@ <h1>Login</h1>
            
             <input class="checkbox" type="checkbox">Remember me</input>
```
Comment (irajbhinde) [inline index.html:47]: Increase Remember me checkbox size for better visibility

File: index.html [hunk 6]
```diff
@@ -45,7 +46,7 @@ <h1>Login</h1>
...
             <a class="forget-password" href="/">Forget Your Password?</a>
-            <button class="btn btn-primary"><a class="login-btn" href="/home/index.html">Log in</a></button>
+            <button class="click-me-link-btn new-ac"><a href="/home/index.html">Log in</a></button>
             <a class="new-account" href="/signup/index.html">Create New Account<i class="fas fa-chevron-right"></i></a>
```
Comment (irajbhinde) [inline index.html:50]: add a little padding between create new account & >

File: loginstyles.css [hunk 7]
```diff
@@ -1,92 +1,78 @@
...
+    width: 35rem;
     border-radius: 10px;
     align-items: center;
-    padding: 40px 70px;
+    padding: 3rem 6rem;
 
     box-shadow: 2px -3px 13px -3px rgba(156, 147, 147, 0.94);
     -webkit-box-shadow: 2px -3px 13px -3px rgba(156, 147, 147, 0.94);
     -moz-box-shadow: 2px -3px 13px -3px rgba(156, 147, 147, 0.94);
 }
```
Comment (HarshnaNHaswani) [inline loginstyles.css:41]: use same format for colors, put them in variables. If you want transparency you can add two digits at the end of hex for that. e.g. #rrggbbaa - #66fa99b3

File: index.html [hunk 8]
```diff
@@ -19,17 +19,18 @@
...
         <link rel="stylesheet" href="loginstyles.css" />
         <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://neo-ui.netlify.app/styles.css">
         
     </head>
     <body>
-        <nav class="navbar navbar-white bg-white">
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
```
Comment (irajbhinde) [inline index.html:27]: Add an anchor tag here with your landing page link

File: loginstyles.css (hunk 1 above)
Comment (poojakelkar) [reply loginstyles.css:11]: Yeah, I have already done that.

File: loginstyles.css [hunk 9]
```diff
@@ -1,92 +1,78 @@
 /* Navbar Design */
+:root{
+    --heart-icon-color: #f10000;
+    --background-color: #ffff;
+    --light-white: #e9e9e9;
+    --font-dark-black-color: #111;
+    --grey-color-font: #888;
+    --button-color: #0031D1;
+}
 body {
```
Comment (irajbhinde) [inline loginstyles.css:10]: You can add CSS RESET here

File: index.html (hunk 4 above)
Comment (poojakelkar) [reply index.html:33]: Sure, Thanks

**Review by mohitdhatrak — COMMENTED** [review]
I guess you made few changes already, Raj covered few points too. Well done1

**Review by irajbhinde — COMMENTED** [review]
Hey! Kindly look into the feedback. Another feedback would be to use a different bg color for nav & main content. As Vikram sir said, avoid using pure black & pure white, use shades of white & black

**Comment by HarshnaNHaswani** [comment]
image
This can be signup instead, in login page
Rest looks good. There is consistency in units everywhere

File: index.html (hunk 5 above)
Comment (poojakelkar) [reply index.html:47]: Thank u

File: index.html (hunk 8 above)
Comment (poojakelkar) [reply index.html:27]: I have added an anchor tag in the login button.

File: loginstyles.css (hunk 9 above)
Comment (poojakelkar) [reply loginstyles.css:10]: SureThanks for the suggestion @irajbhinde

File: login.html (hunk 3 above)
Comment (poojakelkar) [reply login.html:49]: I forget to change the name Thanks @HarshnaNHaswani

File: login.html (hunk 2 above)
Comment (poojakelkar) [reply login.html:31]: Yeah I have already changed the class name

File: loginstyles.css (hunk 7 above)
Comment (poojakelkar) [reply loginstyles.css:41]: Ok I will check thanks

**Pull request merged**